src/routes/tokens/components/ApiToken.tsx

```diff
@@ -0,0 +1,178 @@
+                                fontWeight="400"
+                            >
+                                <Text>
+                                    {'Expire' +
+                                        (tokenIsExpired ? 'd' : 's') +
+                                        ` on ${
+                                            shortFormatDate(
+                                                apiToken.expires_at.toDateString()
+                                            ).split(',')[0]
+                                        } `}
+                                </Text>
+                                <Text opacity={isHovered ? '100%' : '50%'}>
+                                    {` at ${apiToken.expires_at.toLocaleTimeString()}`}
+                                </Text>
+                            </HStack>
+                        )}
+                    </VStack>
+                    <Spacer />
+                    <Center width="120px">
+                        <Button
+                            onClick={onConfirmOpen}
+                            variant="outline"
+                            size="sm"
+                            fontWeight="600"
+                        >
+                            Delete
+                        </Button>
+
+                        <AlertDialog
+                            isOpen={isConfirmOpen}
+                            leastDestructiveRef={cancelRef}
+                            onClose={onConfirmClose}
+                            size="sm"
+                        >
+                            <AlertDialogOverlay>
+                                <AlertDialogContent fontSize="sm">
+                                    <AlertDialogHeader
+                                        fontSize="lg"
+                                        fontWeight="bold"
+                                    >
+                                        Delete token
+                                    </AlertDialogHeader>
+                                    <AlertDialogBody>
+                                        Are you sure you want to delete this
+                                        token?
+                                    </AlertDialogBody>
+                                    <AlertDialogFooter>
+                                        <Button
+                                            onClick={onConfirmClose}
+                                            size="sm"
+                                        >
+                                            Cancel
+                                        </Button>
+                                        <Button
+                                            colorScheme="red"
+                                            onClick={revokeToken}
+                                            ml="3"
+                                            size="sm"
+                                        >
+                                            Yes I'm sure
```

Comment:
I defer to David on this because I have very little experience in UX, but it seems to me it's better to label buttons with the action clicking them will cause (because people don't read the message and jump to the button); so maybe it should be "Delete token".